kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -333,6 +336,10 @@ define([
                             hide: Config.get('tooltip').hideDelay
                         }
                     });
+                    $('td:eq(1)', nRow).find('span.kb-data-staging-folder').off('click').on('click', e => {
```

Comment:
Datatables is finicky. I'm not sure if I'm assigning click events to it incorrectly or not, but it updates and re-runs these steps whenever a sort function is added. Which means that this would trigger those clicks an extra time for each time you sort. Including an `off('click')` first removes any extra events.